src/deploy_inst.cc

```diff
@@ -17,9 +17,10 @@ void DeployInst::Build(cyclus::Agent* parent) {
   std::set<std::string> protos;
   for (int i = 0; i < prototypes.size(); i++) {
     std::string proto = prototypes[i];
+    std::string inst = this->prototype();
 
     std::stringstream ss;
-    ss << proto;
+    ss << proto << "_" << inst << "_";
```

Comment:
I don't think you want the trailing underscore
```suggestion
    ss << proto << "_" << inst;
```